config/initializers/blacklight_initializer.rb

```diff
@@ -65,7 +66,14 @@ def export_as_openurl_ctx_kev(format = nil)
         export_text << "&amp;rft.isbn=#{(isbn.nil? or isbn['a'].nil?) ? "" : isbn['a']}"
       end
     end
-    export_text << "&amp;rft_id=" + (id.nil? ? "" : CGI::escape("pul_"+id.value))
+    
+    export_text << "&amp;rft_id=" + (id.nil? ? "" : CGI::escape("http://bibdata.princeton.edu/bibliographic/"+id.value))
```

Comment:
rft_id is catchall parameter in OpenURL that can be repeated that is supposed to contain either a reference to an identifier like a DOI, LCCN, or OCLCnum or just a basic encoded URL for the resource in question. It is up to the OpenURL resolver to do something with them.